hbase-server/src/main/java/org/apache/hadoop/hbase/coprocessor/RegionObserver.java

```diff
@@ -1567,7 +1567,8 @@ default List<Pair<Cell, Cell>> postAppendBeforeWAL(
    * @param ctx        the environment provided by the region server
    * @param delTracker the deleteTracker that is created by the QueryMatcher
    * @return the Delete Tracker
-   * @deprecated Since 2.0 with out any replacement and will be removed in 3.0
+   * @deprecated Since 2.0, will be removed in 4.0. Visibility label feature still use this method,
```

Comment:
Please change to '2.0.0', '4.0.0' and '3.0.0'

Reply:
@Apache9  Thanks for your review. I have revised it. could you take a look? Thanks.